tldr/cfg_extractor.py

```
@@ -831,6 +958,129 @@ def _visit_ruby_call(self, node):
+        for child in node.children:
+            if child.type == "arguments":
+                arguments_node = child
+                break
+
+        if arguments_node:
+            # Skip first argument (the switch expression)
+            # Process remaining arguments as cases
+            arg_children = [c for c in arguments_node.children if c.is_named]
+            case_args = arg_children[1:] if len(arg_children) > 1 else []
+
+            for i, case_arg in enumerate(case_args):
```

Ruff (0.14.13)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)